package.json

```diff
@@ -54,7 +54,7 @@
     "ember-interpolate-helper": "1.0.0",
     "ember-keyboard": "2.1.10",
     "ember-load-initializers": "^1.0.0",
-    "ember-modal-dialog": "0.9.0",
+    "ember-modal-dialog": "^2.4.4",
```

Comment:
Bumping two major versions without testing this is all platforms before merging seems risky...

Reply:
Was tested on search and offer page. There is only one breaking change (https://github.com/yapplabs/ember-modal-dialog/pull/169) and we're not impacted by it.